src/senselab/audio/tasks/voice_cloning/api.py

```diff
@@ -17,27 +15,39 @@ def clone_voices(
     device: Optional[DeviceType] = None,
     **kwargs: Any,  # noqa:ANN401
 ) -> List[Audio]:
-    """Clones voices from source audios to target audios using the given model."""
+    """Clones voices from source audios to target audios using the given model.
```

Comment:
I think we can be a little clearer on exactly what is happening. That this does a pairwise conversion from each audio in the source list onto the corresponding audio in the target list.